api/controllers/votes.controller.js

```diff
@@ -0,0 +1,235 @@
+import {logger} from "../../modules/Logger.js";
+import {DebugSaveToFile} from "../../modules/DebugMode.js";
+import fs from "fs";
+import {getVotesData, votesPath} from "../../modules/VotesConnector.js";
+import {downloader, getTrackInfo} from "../../modules/MusicDownloader.js";
+import path from "path";
+import {checkIfVLConVotes} from "../../modules/Other.js";
+import {killPlayerForce} from "../../modules/MusicPlayer.js";
+
+export async function getVotes(req, res) {
+    try {
+        if (!fs.existsSync(votesPath)) return res.status(500).send('Brak pliku!!');
+        const data = JSON.parse(fs.readFileSync(votesPath));
+        return res.status(200).send(data);
+    } catch (e) {
+        logger('verbose', 'Wystąpił błąd podczas pobierania głosów', 'LocalAPI - getVotes');
+        if (global.debugmode === true) {
+            DebugSaveToFile('LocalAPI', 'getVotes', 'catched_error', e);
+            logger('verbose', `Stacktrace został zrzucony do debug/`, 'LocalAPI - getVotes');
+        }
+    }
+}
+
+export async function delVotes(req, res) {
+    try {
+        const { id } = req.params;
+        if (!fs.existsSync(votesPath)) return res.status(500).send('Brak pliku!!');
+        const data = JSON.parse(fs.readFileSync(votesPath));
+
+        const index = data.findIndex(vote => vote.id === parseInt(id));
+        if (index === -1) {
+            return res.status(404).send('Głos nie znaleziony!');
+        }
+        data.splice(index, 1);
+        fs.writeFileSync(votesPath, JSON.stringify(data, null, 2));
+
+        return res.status(200).send('Głos usunięty');
+    } catch (e) {
+        logger('verbose', 'Wystąpił błąd podczas usuwania głosu', 'LocalAPI - setVotes');
```

Comment:
```suggestion
        logger('verbose', 'Wystąpił błąd podczas usuwania głosu', 'LocalAPI - delVotes');
```
Coś się panu zapomniało zmienić